spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkScan.java

```diff
@@ -193,15 +202,41 @@ public boolean supportColumnarReads(InputPartition partition) {
     }
   }
 
+  static long numFilesToScan(CombinedScanTask scanTask) {
```

Comment:
numSplitsToScan?

Reply:
Since this is now a one-liner, I have removed the function altogether.